ws-client/src/client.rs

```diff
@@ -251,153 +261,129 @@ async fn background_task(
 		let next_backend = backend_event.next();
 		futures::pin_mut!(next_frontend, next_backend);
 
-		futures::select! {
```

Comment:
changed to `futures::select` to improve readability, reduces one level of indentation.